tools/genArgs.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
+import sys
+import os
+
+def genArgs( fileName, api, includes ):
+    f = open(fileName)
```

Comment:
Suggest to use the `with` keyword when working with unmanaged resources

```suggestion
     with open(fileName) as f:
```
Otherwise `f` is not closed  (`f.close()`)

Reply:
fixed